hail/src/main/scala/is/hail/table/Table.scala

```diff
@@ -28,14 +29,18 @@ case object Descending extends SortOrder
 
 case class SortField(field: String, sortOrder: SortOrder)
 
+abstract class AbstractTableSpec extends RelationalSpec {
```

Comment:
Do you want RelationalSpec to be AbstractRelationalSpec, since it's also abstract?

Reply:
meh. I like relational spec